rest-java/src/main/java/org/hiero/mirror/restjava/util/BytesUtil.java

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.restjava.util;
+
+public final class BytesUtil {
+
+    private BytesUtil() {
+        throw new UnsupportedOperationException("Utility class cannot be instantiated");
+    }
+
+    public static byte[] incrementByteArray(byte[] bytes) {
+        byte[] result = bytes.clone();
+
+        for (int i = result.length - 1; i >= 0; i--) {
+            int v = (result[i] & 0xFF) + 1;
+            result[i] = (byte) v;
+
+            if (v <= 0xFF) {
+                break;
+            }
+
+            result[i] = 0;
+        }
```

Comment:
I think the logic is correct, but it's inefficient to set result[i] twice. Let's make sure a test gets added.

```suggestion
        for (int i = bytes.length - 1; i >= 0; --i) {
            if (bytes[i] == (byte) 0xFF) {
                bytes[i] = (byte) 0x00;
            } else {
                ++bytes[i];
                break;
            }
        }
```

Same sort of thing for decrement but swap FF and 00.